### ISSUE TITLE
v2.1.15  impossible to submit shipments

### ISSUE THREAD

**Issue by Peter-Liska** [Zasilkovna/prestashop#215]
It is currently impossible to submit shipments with v2.1.15 (Prestashop 1.7.7.8)

And all orders have disappeared when updated to latest version (updated from v2.1.14, which worked well).

**Comment by Peter-Liska** [comment]
After manually loading the module, then it once again requested the same update via Module Management (2.1.14 -> 2.1.15) and it looks like it started working. Strange.

**Comment by MatyasKoval** [comment]
Hello @Peter-Liska, 
thank you very much for your comment, we were just getting into trying to replicate your issue. 

Could you please describe to us the precise way how you got this issue, so we could replicate it, find the cause of it and solve it?

Best regards,
Matyáš Kovaľ - Packeta E-Commerce Support

**Comment by Peter-Liska** [comment]
Downloaded the module v.2.1.5 from [releases](https://github.com/Zasilkovna/prestashop/releases), then uploaded it to Prestashop. Update completed without warning. And then it stopped working.

Subsequently, a request for automatic update appeared in Module Management (2.1.14 -> 2.1.15) 
Updated once again and it started working. Strange.

**Comment by MatyasKoval** [comment]
Hello @Peter-Liska ,
thank you kindly for the fast answer.

We have not encountered this issue during our testing. 
We will definitely look into this to try and replicate this behaviour and let you know what we come up with.

Best regards,
Matyáš Kovaľ - Packeta E-Commerce Support

**Comment by MatyasKoval** [comment]
Hello @Peter-Liska,
We have tried extensively to replicate this issue when updating from different versions to 2.1.15, but in all cases, the update went through smoothly and without this behaviour happening. 
Without being able to replicate this we can not really determine the cause (nor can we completely rule out that this may have been caused by something unrelated to our module), as your case was the only one we have encountered so far.

Thank you none the less for your feedback.
Best regards,
Matyáš Kovaľ - Packeta E-Commerce Support
